portal.h

```diff
@@ -62,6 +57,62 @@ void setSSID(String ssid){
   return;
 }
 
+void confirmOrTypeSSID(){
+  DISP.fillScreen(BLACK);
+  DISP.setSwapBytes(true);
+  DISP.setTextSize(MEDIUM_TEXT);
+  DISP.setTextColor(TFT_RED, BGCOLOR);
+  DISP.setCursor(0, 0);
+  DISP.println("WiFi SSID");
+  DISP.setTextSize(TINY_TEXT);
+  DISP.setTextColor(FGCOLOR, BGCOLOR);
+  DISP.println(TYPE_SSID_TEXT);
+  DISP.setTextSize(SMALL_TEXT);
+  uint8_t ssidTextCursorY = DISP.getCursorY();
+  String currentSSID = String(apSsidName.c_str());
+  DISP.printf("%s", currentSSID.c_str());
+  bool ssid_ok = false;
+
+  while(!ssid_ok){
+    M5Cardputer.update();
+    if(M5Cardputer.Keyboard.isChange() && M5Cardputer.Keyboard.isPressed()) {
+      Keyboard_Class::KeysState status = M5Cardputer.Keyboard.keysState();
+      for(auto i : status.word) {
+        if(i != '?' && i != '$' && i != '\"' && i != '[' && i != '\\' && i != ']' && i != '+'){
+          currentSSID += i;
+        }
+      }
+      if(currentSSID.length() >= 32) {
+        continue;
+      }
+      if(status.del) {
+        currentSSID.remove(currentSSID.length() - 1);
+      }
+      if(status.enter) {
+        ssid_ok = true;
+      }
+      DISP.fillRect(0, ssidTextCursorY, DISP.width(), DISP.width()- ssidTextCursorY, BLACK);
+      DISP.setCursor(0, ssidTextCursorY);
+      DISP.printf("%s", currentSSID.c_str());
+    }
+    delay(100);
+  }
+
+  if(currentSSID.length() > 2){
+    setSSID(currentSSID);
```

Comment:
This will write to EEPROM every single time you start NEMO portal, causing excess EEPROM wear. Additionally, this will cause which ever WiFi you clone from WiFi scanner to be the new default SSID for NEMO Portal even if you don't want it to be. 

Two changes I would like: 
1) Confirm y/n if the user wishes this SSID to be saved to EEPROM
1) If the SSID in EEPROM matches the one to be saved, skip writing the EEPROM to keep from excess EEPROM cycles